packages/subcommands/package.json

```diff
@@ -0,0 +1,59 @@
+{
+	"name": "@sapphire/subcommands",
+	"version": "0.0.1",
+	"description": "Plugin for @sapphire/framework that adds support for subcommands.",
+	"author": "@sapphire",
+	"license": "MIT",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"exports": {
+		".": {
+			"import": "./dist/index.mjs",
+			"require": "./dist/index.js"
+		}
+	},
+	"homepage": "https://github.com/sapphire-project/plugins/tree/main/packages/subcommands",
+	"scripts": {
+		"lint": "eslint src tests --ext ts --fix",
+		"build": "tsc -b src",
+		"postbuild": "run-p esm:**",
+		"esm:register": "gen-esm-wrapper dist/register.js dist/register.mjs",
+		"esm:default": "gen-esm-wrapper dist/index.js dist/index.mjs",
```

Comment:
```suggestion
		"postbuild": "gen-esm-wrapper dist/index.js dist/index.mjs",
```

Reply:
Resolved in 6ea0a87491b4571304bf8069df488230e4a2b379